test/image/mocks/sankey_x_y.json

```diff
@@ -0,0 +1,28 @@
+{
+    "data": [
+        {
+            "type": "sankey",
+            "arrangement": "freeform",
```

Comment:
Do all the potential values of `arrangement` have an effect of the node layout when `node.x` and `node.y` are set?

Reply:
> Do all the possible values of arrangement have an effect of the node layout when node.x and node.y are set?

Short answer: only `snap` can override `node.(x|y)`

Long answer:
When `node.(x|y)` is set, the nodes are forced to sit at the specified locations except in the `snap` arrangement. When drag and dropping, `snap` is the only arrangement in which we forbid nodes from overlapping so I made sure we keep honoring this promise.

The goal of this PR was to make it possible to replicate any state accessible via drag and drop by setting `node.(x|y)` and hence allowing to save to export images.

Reply:
Ok, thanks for the info!

Could we then make `arrangement` default to `freeform` whenever `node.(x|y)` are set? That way, a user that wants to programmatically tweak the node position via `node.(x|y)` can do so w/o having to worry about changing the `arrangement` value.

Reply:
Done in 65c7201